akka-docs/rst/scala/persistence-schema-evolution.rst

```diff
@@ -0,0 +1,386 @@
+.. _persistence-schema-evolution-scala:
+
+##############################
+Persistence - Schema Evolution
+##############################
+
+When working on long running projects using :ref:`persistence-scala`, or any kind of `Event Sourcing`_ architectures,
+schema evolution becomes one of the more important technical aspects of developing your application since both
+requirements as well as our own understanding of the business domain may (and will) change in time.
```

Comment:
perhaps split this sentence after “application”; the “both” also disturbs the reading flow a little—perhaps “The requirements will change as well as our understanding …”